package.json

```diff
@@ -20,20 +20,28 @@
     "update-ts-references": "update-ts-references"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.3",
+    "@octokit/core": "^3.2.1",
+    "@storybook/react": "^6.0.28",
+    "@testing-library/react": "^11.1.2",
     "@types/jest": "^26.0.15",
-    "@typescript-eslint/eslint-plugin": "^4.6.1",
-    "@typescript-eslint/parser": "^4.6.1",
+    "@types/react": "^16.9.56",
+    "@typescript-eslint/eslint-plugin": "^4.8.1",
+    "@typescript-eslint/parser": "^4.8.1",
     "eslint": "^7.12.1",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^24.1.0",
+    "eslint-plugin-jest": "^24.1.3",
     "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-testing-library": "^3.10.0",
     "jest": "^26.6.3",
     "lerna": "^3.22.1",
     "prettier": "^2.1.2",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
```

Comment:
React and react-dom are peer deps of storybook/react. We do use React in the prepareStory tests, though.